jemoji.gemspec

```diff
@@ -10,7 +10,7 @@ Gem::Specification.new do |s|
   s.files    = [ 'lib/jemoji.rb' ]
 
   s.add_dependency 'jekyll', '>= 2.0'
-  s.add_dependency 'html-pipeline', '~> 1.9'
+  s.add_dependency 'html-pipeline', '~> 2.2.2'
```

Comment:
This won't allow us to use 2.3 or higher. Is that intended? This says `>= 2.2.2, < 2.3.0`